event-streams/Dockerfile

```diff
@@ -13,6 +13,7 @@ ADD event-streams /src
 ADD schema/run.event.yaml /src
 ADD schema/run.host.event.yaml /src
 
+RUN microdnf update -y && microdnf install jq
 RUN microdnf install gzip
```

Comment:
Would it be possible to combine these two lines?

```
RUN microdnf update -y && microdnf install jq gzip
```

I think each command ends up being a new layer (more to download, etc).

Reply:
done, thanks!